chpl/chpl-service/src/main/java/gov/healthit/chpl/SpringContext.java

```diff
@@ -26,4 +33,16 @@ public void setApplicationContext(ApplicationContext appContext) throws BeansExc
         // store ApplicationContext reference to access required beans later on
         SpringContext.context = appContext;
     }
+
+    @Override
+    protected Object createJobInstance(final TriggerFiredBundle bundle) throws Exception {
```

Comment:
I was just wondering if you know why this would be needed in both the QuartzJobFactory and here in the SpringContext. No issue if it is necessary, it just looks like the same thing declared in 2 places.